### DISCUSSION TITLE
Can't show/pull: The unauthenticated git protocol on port 9418 is no longer supported

### DISCUSSION THREAD

**Discussion by TimAlderson** [community/community#21987] [New to GitHub]
<p>I am not looking to push, only to read and pull to update my local version.</p>
<p>All worked fine up to a month ago, but now I get the following</p>
<p>git remote show origin<br>
fatal: remote error:<br>
The unauthenticated git protocol on port 9418 is no longer supported.<br>
Please see <a href="https://github.blog/2021-09-01-improving-git-protocol-security-github/" class="inline-onebox" rel="noopener nofollow ugc">Improving Git protocol security on GitHub | The GitHub Blog</a> for more information.</p>
<p>I read through the info via url, and nothing seems to apply.</p>
<p>Running Ubuntu 20.04,  OpenSSH_8.2p1 Ubuntu-4ubuntu0.4, OpenSSL 1.1.1f  31 Mar 2020</p>
<p>Putty version is 0.76</p>
<p>URL’s are https</p>
<p>/opt/webwork/webwork2/.git$ git remote -v<br>
origin  <a href="https://github.com/openwebwork/webwork2.github.com.git" rel="noopener nofollow ugc">https://github.com/openwebwork/webwork2.github.com.git</a> (fetch)<br>
origin  <a href="https://github.com/openwebwork/webwork2.github.com.git" rel="noopener nofollow ugc">https://github.com/openwebwork/webwork2.github.com.git</a> (push)</p>
<p>Any ideas what I should look into next?</p>

**Comment by airtower-luna** [comment]
<p>Check your Git config if there is any <a href="https://git-scm.com/docs/git-config#Documentation/git-config.txt-urlltbasegtinsteadOf"><code>insteadOf</code> setting</a> that might match those URLs, and make Git actually use something else.</p>

**Comment by TimAlderson** [comment]
<p>contents of .gitconfig</p>
<p>[url “<a href="https://github.com/" rel="noopener nofollow ugc">https://github.com/</a>”]<br>
insteadOf = git://github.com/</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>That’s your problem right there, your configuration tells Git to use the plaintext Git protocol instead of HTTPS. Remove those lines and it will use HTTPS.</p>

**Comment by TimAlderson** [comment]
<p>That did it!</p>
<p>Thank you so much for taking the time to help. Truly appreciated.</p>

**Comment by TimAlderson** [comment]
<p>Contents of config in my .git directory:</p>
<p>[core]<br>
repositoryformatversion = 0<br>
filemode = true<br>
bare = false<br>
logallrefupdates = true<br>
[branch “release/2.10”]<br>
remote = .<br>
merge = refs/heads/master<br>
[remote “origin”]<br>
url = <a href="https://github.com/openwebwork/webwork2.git" class="inline-onebox" rel="noopener nofollow ugc">GitHub - openwebwork/webwork2: Course management front end for WeBWorK</a><br>
fetch = +refs/heads/<em>:refs/remotes/origin/</em></p>

**Comment by gfishlab** [comment]
<p>Continuing the discussion from <a href="https://github.community/t/cant-show-pull-the-unauthenticated-git-protocol-on-port-9418-is-no-longer-supported/242962/6">Can't show/pull: The unauthenticated git protocol on port 9418 is no longer supported</a>:</p>
<p>I have done these steps ,but this do not work.Can you help me look at? Thanks!,<br>
Now I can only pull remote code By SSH, instead of HTTPS.<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/2/1/21d5a47c9985e5851ce5f08979849309665ca6ef.png" data-download-href="/uploads/short-url/4PjtYFQHtSHeV5SgpzCwLux1pXV.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/49182687/181068049-523034ee-15fe-4c4e-8edb-ef31eb7383ee.png" alt="image" data-base62-sha1="4PjtYFQHtSHeV5SgpzCwLux1pXV" width="690" height="151" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/2/1/21d5a47c9985e5851ce5f08979849309665ca6ef_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1000×220 11.7 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by airtower-luna** [comment]
<aside class="quote no-group" data-username="gxj1134506645" data-post="7" data-topic="242962">
<div class="title">
<div class="quote-controls"></div>
 gxj1134506645:</div>
<blockquote>
<p>I have done these steps ,but this do not work.</p>
</blockquote>
</aside>
<p>Which steps have you done, and what exactly is the goal you’re trying to achieve? In the screenshot it looks like you tried to <em>create</em> a setting that should make Git use SSH instead of HTTPS, but it actually uses plaintext Git protocol.</p>
<p>You’ll have to check your configuration, <a href="https://git-scm.com/docs/git-config#FILES" class="inline-onebox">Git - git-config Documentation</a> describes where it can be (there are a bunch of locations).</p>
